### ISSUE TITLE
Validation result

### ISSUE THREAD

**Issue by zou-yawen** [vimar-gu/MinimaxDiffusion#6]
hi, I repeat the same validation code with the same generation images, it has different results every time. I'm a little confused.For example, the same woof ipc 10 images， I repeat the same validation code, it has 38.2,36.4,37.0 respectively.Could you please help me

**Comment by vimar-gu** [comment]
Hi, we adopt the same validation protocol as IDC. Indeed it yields different results each time, so we report the mean and standard variation for the tables in the paper.

**Comment by zou-yawen** [comment]
> Hi, we adopt the same validation protocol as IDC. Indeed it yields different results each time, so we report the mean and standard variation for the tables in the paper.
Thank you for your reply, you use three different seeds to generate images and images produced by each seed are validated three times, then report the mean and standard variation?

**Comment by vimar-gu** [comment]
You can set the `--repeat` parameter, then the script yields the mean and standard variation automatically after the desired runs.

**Comment by zou-yawen** [comment]
> You can set the `--repeat` parameter, then the script yields the mean and standard variation automatically after the desired r

**Comment by zou-yawen** [comment]
> > You can set the `--repeat` parameter, then the script yields the mean and standard variation automatically after the desired r
Thank you very much, I will try it.

**Comment by zou-yawen** [comment]
hello, sorry to bother you again. I am a little confused. In the GitHub code, the woof data  use the val data to calculate the accuracy; 
`elif args.dataset == 'imagenet':
        if tsne:
            val_subdir = 'train'
        else:
            val_subdir = 'val'
        if len(args.imagenet_dir) == 1:
            traindir = os.path.join(args.imagenet_dir[0], 'train')
            valdir = os.path.join(args.imagenet_dir[0], val_subdir)
        else:
            traindir = args.imagenet_dir[0]
            valdir = os.path.join(args.imagenet_dir[1], val_subdir)`
in your paper, you said '''The top-1 accuracy on the
original testing set is reported to illustrate the performance.'''.

**Comment by zou-yawen** [comment]
hello, sorry to bother you again. I am a little confused. In the GitHub code, the woof data  use the val data to calculate the accuracy; 
`elif args.dataset == 'imagenet':
        if tsne:
            val_subdir = 'train'
        else:
            val_subdir = 'val'
        if len(args.imagenet_dir) == 1:
            traindir = os.path.join(args.imagenet_dir[0], 'train')
            valdir = os.path.join(args.imagenet_dir[0], val_subdir)
        else:
            traindir = args.imagenet_dir[0]
            valdir = os.path.join(args.imagenet_dir[1], val_subdir)`
in your paper, you said '''The top-1 accuracy on the
original testing set is reported to illustrate the performance.'''.

**Comment by vimar-gu** [comment]
Thanks for the question. Sorry about the confusion. People are actually always using the validation subset for testing. I should have used the "validation set" in the manuscript, but I was actually referring to the validation set with the "testing set".

**Comment by zou-yawen** [comment]
> Thanks for the question. Sorry about the confusion. People are actually always using the validation subset for testing. I should have used the "validation set" in the manuscript, but I was actually referring to the validation set with the "testing set".

Thanks for your reply.

**Comment by zou-yawen** [comment]
hi, sorry to bother you again. Could you please give me the ImageIDC class list, I didn't find it. For  ImageIDC you use ResNet-10 with average pooling, how do you run in your code for validation?

**Comment by vimar-gu** [comment]
Hi, it is described in the IDC paper, and is the first 10 classes of ImageNet-100. The validation process is the same as the other datasets.

**Comment by zou-yawen** [comment]
> Hi, it is described in the IDC paper, and is the first 10 classes of ImageNet-100. The validation process is the same as the other datasets.
Thank you very much

**Comment by zou-yawen** [comment]
> Hi, it is described in the IDC paper, and is the first 10 classes of ImageNet-100. The validation process is the same as the other datasets.
hello, sorry to bother you again. I am a little confused. .In the IDC paper the  ImageIDC  is validate by  ResNetAP-10; in your paper 
![image](https://github.com/user-attachments/assets/ea086b88-5fe0-47ba-a13e-c7f65af75071)
this result is also come from ResNetAP-10?
there two places mentioned about this, whether in your paper ResNetAP-10 equal ResNet-10
“”“ResNet-10  with average pooling is adopted for matching the feature distribution
(DM , GLaD ) and training gradients (IDC-1 ).”“”
“”“ResNetAP-10 is a 10-layer ResNet , where the
strided convolution is replaced by average pooling for
downsampling”“”
